mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -132,7 +132,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_1521 reloadResources ()Ljava/util/concurrent/CompletableFuture;
 	METHOD method_1522 getFramebuffer ()Lnet/minecraft/class_276;
 	METHOD method_1523 render (Z)V
-		ARG 1 fullRender
+		ARG 1 tick
```

Comment:
`inTick` or `ticking`? `tick` is not very specific here in my opinion.
```suggestion
		ARG 1 inTick
```

Reply:
No, if `tick` is false, then it will render a frame without ticking the world.

Reply:
i suggest ticking then

Reply:
It can be false even when the client is ticking (for example, it's called by the dimension change method to do a quick render without ticking the world).

Reply:
Possibly something like `doTick` or `shouldTick` could work?

Reply:
i feel a name like countToFps may be more clear?